modules/dapps.js

```diff
@@ -141,7 +141,7 @@ __private.list = function(filter, cb) {
 	if (filter.category) {
 		const category = dappCategories[filter.category];
 
-		if (category != null) {
+		if (category !== null) {
```

Comment:
`if (category)`?

Reply:
No, because if category is `0` which is different than `null` it won't pass the control.

Reply:
ahh ok 👍